tests/test_tztrout.py

```diff
@@ -64,6 +64,8 @@ class TestTZIdsForPhone:
         [
             ('+1 (650) 333 4444', ['America/Los_Angeles']),
             ('+48 601 941 311', ['Europe/Warsaw']),
+            ('+73519234567', ['Asia/Yekaterinburg']),
+            ('+888569', []),
```

Comment:
I wonder if we should add a lot more parameters here to make sure this is equally (or more) accurate as the previous implementation...

So far we have a lot of tests for US phone numbers matching a specific TZ name (PT/ET/etc.), but that's quite coarse. I'd wanna make sure that most US numbers resolve to a precise TZ ID. Feel free to borrow numbers from the `test_major_cities_us_ca` test.

Another number type we could test is e.g. regionless toll-free numbers in the US. AFAICT they should return *all* US TZ IDs in the list.

Reply:
I did it in #45 and it uncovered some minor differences. I will try to identify if it's our data that's wrong or phonenumbers. Any help would be appreciated. To test phonenumbers, you can use the google's official demo https://libphonenumber.appspot.com/